src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/util/TestStorageHelper.java

```diff
@@ -0,0 +1,109 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.util;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.CloudBlobContainer;
+import com.microsoft.azure.storage.blob.CloudBlockBlob;
+import org.testcontainers.containers.DockerComposeContainer;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.security.InvalidKeyException;
+
+import static uk.gov.hmcts.reform.bulkscanprocessor.helper.DirectoryZipper.zipDir;
+
+public class TestStorageHelper {
+
+    private static TestStorageHelper INSTANCE;
+
+    public static final String CONTAINER_NAME = "bulkscan";
+    public static final String ZIP_FILE_NAME = "1_24-06-2018-00-00-00.zip";
+
+    private static DockerComposeContainer<?> dockerContainer;
```

Comment:
do we really need generics here ? can be just `DockerComposeContainer`

Reply:
Just solving the warning here ;). Class is generic